.NET/Microsoft.Recognizers.Text.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2026
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30128.74
```

Comment:
Be careful not to change the solution configuration unless essential.

Reply:
sorry, pushed by mistake, reverted.